google/cloud/bigtable/admin/integration_tests/table_admin_integration_test.cc

```diff
@@ -349,15 +451,29 @@ TEST_F(TableAdminIntegrationTest, CreateListGetDeleteTableWithLogging) {
   EXPECT_EQ(1, count_matching_families(*table_detailed, "foo"));
 
   // Update table
-  std::vector<bigtable::ColumnFamilyModification> column_modification_list = {
-      bigtable::ColumnFamilyModification::Create(
-          "newfam", GC::Intersection(GC::MaxAge(std::chrono::hours(7 * 24)),
-                                     GC::MaxNumVersions(1))),
-      bigtable::ColumnFamilyModification::Update("fam", GC::MaxNumVersions(2)),
-      bigtable::ColumnFamilyModification::Drop("foo")};
-
-  auto table_modified =
-      table_admin->ModifyColumnFamilies(table_id, column_modification_list);
+  using Mod = btadmin::ModifyColumnFamiliesRequest::Modification;
```

Comment:
nit: `using btadmin::ModifyColumnFamiliesRequest::Modification;` seems like would work?

Reply:
Apparently no, because `Modification` is not a class. The generated proto code is:
```c++
typedef ModifyColumnFamiliesRequest_Modification Modification;
```
I could save a line (in two places) by just saying:
```c++
std::vector<btadmin::ModifyColumnFamiliesRequest::Modification> mods(3)
```
... but I think that's a bit harder to read